### DISCUSSION TITLE
Single Sign-On for admin panel

### DISCUSSION THREAD

**Discussion by r4fx** [keystonejs/keystone#6600] [Event Q&A Q3 2021]
Is it possible to use Single Sign-On on entering to KS admin panel?

**Comment by bladey** [comment]
Hi @r4fx, we don't have an out of the box solution at the moment, however I recommend you check out `Keystone Next Auth` from @borisno2's which adds social authentication to Keystone 6 here - https://github.com/OpenSaasAU/keystone-nextjs-auth
